astro/src/content/quickstarts/quickstart-python-django-web.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Python Django
```

Comment:
Is this title intentional? If not, I'd prefer we change to "Adding Authentication To A Python And Django App With FusionAuth"

Reply:
@bradmccarty the title of all the quickstarts is just the technology. 

https://fusionauth.io/docs/quickstarts/quickstart-python-flask-web

It is also what shows up on the quickstarts index:

https://fusionauth.io/docs/quickstarts/

I could switch this around for all of the quickstarts if you think it is very important.

Reply:
@mooreds Nope. Not that important. We're ranking well for all of those keywords anyway. Thanks for the clarification.